### ISSUE TITLE
Data Integrity - Category Url Path - error if data integrity is found

### ISSUE THREAD

**Issue by spankevich** [baldwin-agency/magento2-module-url-data-integrity-checker#45]
Error: assert(is_string($category->getUrlPath()))

![Image](https://github.com/user-attachments/assets/934ee3fe-fea3-4c79-95d0-f7baa1351a07)

**Comment by hostep** [comment]
Hey @spankevich 

Sorry for the delay, I was in holiday some weeks ago and am still catching up on certain things that happened while I was away, like this report.

At the moment, I'm not really sure how this can happen, could you maybe try to execute the following on the command line and see if that gives some more info?
```
$ bin/magento catalog:category:integrity:urlpath -vvv
```

I have the suspicion that you have some strange configuration setup in your php.ini config file(s) that maybe cause this issue, so maybe you can also show the output of this command:
```
$ php -i | grep -i assert
```

Thanks!
